src/Editor/GUI/FolderNameWindow.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <functional>
+#include <string>
+
+namespace GUI {
+    /// A window that asks the user for the name of the folder to create.
+    class FolderNameWindow {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The class 'FolderNameWindow' does not have a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoConstructor)